### PR TITLE
Fix IBU rollback script var

### PR DESCRIPTION
**Pull request by omertuc**
See https://github.com/rh-ecosystem-edge/ib-orchestrate-vm/pull/66

### CODE REVIEW

**Review by donpenney — APPROVED** [review]

**Comment by omertuc** [comment]
/pj-rehearse pull-ci-openshift-kni-lifecycle-agent-main-ibu-e2e-flow

**Comment by omertuc** [comment]
/pj-rehearse pull-ci-openshift-kni-lifecycle-agent-main-ibu-e2e-flow

**Comment by omertuc** [comment]
/pj-rehearse pull-ci-openshift-kni-lifecycle-agent-main-ibu-e2e-flow

**Comment by omertuc** [comment]
/pj-rehearse pull-ci-openshift-kni-lifecycle-agent-main-ibu-e2e-flow

File: ci-operator/step-registry/openshift/image-based/upgrade/target/openshift-image-based-upgrade-target-commands.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ echo "Making a target cluster..."
 make target
 
 echo "Upgrading target cluster from ${TARGET_VERSION} to ${SEED_VERSION} using ${SEED_IMAGE}:${SEED_IMAGE_TAG}..."
-make sno-upgrade SEED_IMAGE=${SEED_IMAGE}:${SEED_IMAGE_TAG} DISABLE_IBU_ROLLBACK=true
+make sno-upgrade SEED_IMAGE=${SEED_IMAGE}:${SEED_IMAGE_TAG} DISABLE_IBU_ROLLBACK=Disabled
```
Comment (jerpeter1) [inline ci-operator/step-registry/openshift/image-based/upgrade/target/openshift-image-based-upgrade-target-commands.sh:52]: I think you want `IBU_ROLLBACK=Disabled`

**Comment by omertuc** [comment]
/pj-rehearse pull-ci-openshift-kni-lifecycle-agent-main-ibu-e2e-flow

**Pull request merged**